backend/server.py

```diff
@@ -81,7 +81,12 @@ async def websocket_endpoint(websocket: WebSocket):
                 json_data = json.loads(data[6:])
                 task = json_data["value"].get("task")
                 task_id = int(time.time())
-                
+                base_id = json_data["value"].get("base_id")
+                agent_id = json_data["value"].get("agent_id")
+
+                base_id = base_id if base_id else "STEN2imEGLyHBfz7hgYv28CQ1LjNaC9"
```

Comment:
else case not needed may be, because if user selects Splore Index then these values must come from the UI and if not found should throw an error.

Reply:
Done